pkgs/development/python-modules/pystardict/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, setuptools-scm
+, six
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pystardict";
+  version = "0.8";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "lig";
+    repo = pname;
+    rev = version;
+    hash = "sha256-YrZpIhyxfA3G7rP0SJ+EvzGwAXlne80AYilkj6cIDnA=";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  propagatedBuildInputs = [ six setuptools ];
+
+  nativeBuildInputs = [ setuptools-scm ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "pystardict" ];
+
+  meta = with lib; {
+    description =
+      "Library for manipulating StarDict dictionaries from within Python";
+    homepage = "https://github.com/lig/pystardict";
+    license = licenses.gpl3Only;
```

```suggestion
    license = licenses.gpl3Plus;
```
see, https://github.com/lig/pystardict/blob/ce2bd7b4289411c2ac1536a4d186ade9c3a79aba/setup.py#L9-L10

---

Ahhh that's how you tell. Thanks for linking to direct part. Kept wondering what the difference was